examples/Example/App.hs

```diff
@@ -75,19 +80,57 @@ import Web.Hyperbole.Effect.Server (Request (..))
 run :: IO ()
 run = do
   hSetBuffering stdout LineBuffering
-  putStrLn "Starting Examples on http://localhost:3000"
+
+  port <- do
+    mStr <- SE.lookupEnv "PORT"
+    pure $ fromMaybe 3000 (readMay =<< mStr)
+  putStrLn $ "Starting Examples on http://localhost:" <> show port
+
   users <- Users.initUsers
   count <- runEff $ runConcurrent Counter.initCounter
   cache <- clientCache
-  Warp.run 3000 $
+  Warp.run port $
     Static.staticPolicyWithOptions cache (addBase "client/dist") $
-      Static.staticPolicy (addBase "examples/static") $
-        app users count
+      Static.staticPolicy (addBase "examples/static") $ \req -> do
+        let doc = case pathInfo req of
+              "todoscss" : _ -> toDocumentTodosCSS
+              _ -> toDocument
+        (app doc users count) req
+
+-- Use the embedded version for real applications (see basicDocument).
+-- The link to /hyperbole.js here is just to make local development easier
+toDocument :: BL.ByteString -> BL.ByteString
```

Comment:
No need to branch here. You could either load your custom css on every page by modifying `toDocument`, or, even better, do everything in TodoCSS.page:

```
page :: (Todos :> es) => Eff es (Page '[TodosView, TodoView])
page = do
  todos <- Todos.loadAll
  pure $ tag "div" id $ do
     stylesheet "https://cdn.jsdelivr.net/npm/todomvc-app-css@2.4.3/index.min.css"
     -- the rest of your page
     ...
```

That way, your stylesheet is loaded on your page, and nothing else. All without modifying App.js

Reply:
I branched here because the `toDocument`  function of the example app injects CSS via the head tag, and that would interfere with the CSS loaded after.

![image](https://github.com/user-attachments/assets/40f352af-b646-4c17-895e-7ec10b6c443f)

Reply:
Hmm.... The issue is that Example.App.hs is serving double-duty as an example of how to bootstrap an application. I'd really like to avoid anything as complex as a branch like this if possible. 

This CSS is just a reset. Are you sure it would conflict? If it will definitely mess it up, maybe let's put this in its own app. I'm open to better ideas

Reply:
I went with that assumption but I didn't actually try it. I'll keep you posted.